test/lib/bamboo/adapters/test_adapter_test.exs

```diff
@@ -172,6 +172,69 @@ defmodule Bamboo.TestAdapterTest do
     end
   end
 
+  test "refute_email_delivered_with when email does not match" do
+    mail =
+      new_email(
+        to: to = [{nil, "foo@bar.com"}],
```

Comment:
Not a big deal, but our stylistic preference for this, and `subject` below, is to assign the variable on its own line and then use it, vs. doing it inline like this, since we think it makes it easier to understand at a glance. Your call whether you want to change this!

Reply:
Sure 3928f3e